src/experimental/shader-modules/lighting/index.js

```diff
@@ -1,25 +1,25 @@
-import {readFileSync} from 'fs';
-import {join} from 'path';
 import {Vector3} from '../../../packages/math';
 
-const commonShader = readFileSync(join(__dirname, './lighting-common.glsl'));
-
-export const name = 'lighting';
-
-export const config = {
-  MAX_POINT_LIGHTS: 4
-};
+import commonShader from './lighting-common.glsl';
```

Comment:
variable names: is it better to use more meaningful names here? like `import LightingCommonShader from ''`